privacyidea/api/validate.py

```diff
@@ -198,10 +198,8 @@ def offlinerefill():
                     otps = MachineApplication.get_refill(tokenobj, password, mdef.get("options"))
                     refilltoken = MachineApplication.generate_new_refilltoken(tokenobj)
                     response = send_result(True)
-                    content = json.loads(response.data)
-                    content["auth_items"] = {"offline": [{"refilltoken": refilltoken,
-                                                          "response": otps}]}
-                    response.data = json.dumps(content)
+                    response.json["auth_items"] = {"offline": [{"refilltoken": refilltoken,
```

Comment:
You said [here](https://github.com/privacyidea/privacyidea/pull/1706#issuecomment-507190292) the json attribute would be read-only  - so this should not work, righ!?

And I now realize, why we are using ``.data`` and not ``.json`` (Or am I getting somthing rong here?)

Reply:
Yes, You are right, missed this (and the tests apparently as well)

Reply:
Strange enough, lately I had the impression it *would* be possible. Please check this with @fredreichbier